smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatManager.java

```diff
@@ -82,7 +82,7 @@
     private static final String DISCO_NODE = MUCInitialPresence.NAMESPACE + "#rooms";
 
     private static final Logger LOGGER = Logger.getLogger(MultiUserChatManager.class.getName());
-
+    private final ServiceDiscoveryManager serviceDiscoveryManager;
```

Comment:
The declaration should be not with the block auf static field declarations, but further down where the non-static fields are declared (e.g. just above the first constructor). Also there should be a newline before the next non-empty line comes.